cli/src/vite-plugin/index.js

```diff
@@ -68,14 +68,10 @@ const plugin = (opts) => {
     },
     configureServer(server_) {
       server = server_
-      server.ws.on('connection', () => {
+      server.ws.on('elm:client-ready', () => {
         if (lastErrorSent) {
           server.ws.send('elm:error', {
-            error: ElmErrorJson.toColoredHtmlOutput(lastErrorSent, {
-              GREEN: 'mediumseagreen',
-              RED: 'indianred',
-              BLUE: 'dodgerblue',
-            })
+            error: ElmErrorJson.toColoredHtmlOutput(lastErrorSent),
```

Comment:
I removed the color map here because it's not being used on line 179/175 in this file where errors are also sent and it didn't seem to do anything to the display.  I have a red-green color deficiency though, so I'm not totally certain about that. :grin:

Reply:
Here's the line where the user-defined color palette overrides the default ones– we still want this so that the colors have better contrast on the dark background.

Reply:
Should I also update https://github.com/elm-land/elm-land/blob/main/cli/src/vite-plugin/index.js#L179 to include the color map as well? Or refactor into a shared function used both here and there?

Reply:
Oh, I see what you mean– nice catch! I'm cool approving this as-is, I can follow up with that fix in another PR